scripts/query-profiles.sh

```diff
@@ -1,13 +1,18 @@
 #!/bin/sh
 
+# Unix
+TMP_DIR=/tmp
+# Windows
+# TMP_DIR=/mnt/c/Users/antho/AppData/Local/Temp/
```

Comment:
This is also something to remove. Had it in there because windows apparently makes tmp dirs relative to user home but this isn't typically run on windows.

Reply:
got it! a good idea would be to use WSL's own filesystem, that could make it more general overall, as in `~/`